data/events.yaml

```diff
@@ -0,0 +1,610 @@
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+
+  - name: reset_installation_failed
+    message: "Failed to reset installation. Error: {error}"
+    event_type: cluster
+    severity: "error"
+    properties:
+      cluster_id: UUID
+      error: string
+
+  - name: prepare_installation_failed
+    message: "Failed to prepare the installation due to an unexpected error. Please retry later"
+    event_type: cluster
+    severity: "warning"
+    properties:
+      cluster_id: UUID
+
+  - name: cluster_prepare_installation
+    message: "Cluster was prepared successfully for installation"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+
+  - name: api_ingress_vip_update
+    message: "Cluster was updated with api-vip {api_vip}, ingress-vip {ingress_vip}"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      api_vip: string
+      ingress_vip: string
+
+  - name: cluster_deregistered_after_inactivity
+    message: "Cluster is deregistered due to inactivity"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+
+  - name: cluster_installing_finished
+    message: "Successfully finished installing cluster {cluster_name}"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      cluster_name: string
+
+  - name: cluster_installing_failed
+    message: "Failed installing cluster {cluster_name}. Reason: {failure_reason}"
+    event_type: cluster
+    severity: "critical"
+    properties:
+      cluster_id: UUID
+      cluster_name: string
+      failure_reason: string
+
+  - name: cluster_status_updated
+    message: "Updated status of cluster {cluster_name} to {cluster_status}"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      cluster_name: string
+      cluster_status: string
+
+  - name: installation_preparing_timed_out
+    message: "Preparing for installation was timed out for cluster {cluster_name}"
+    event_type: cluster
+    severity: "warning"
+    properties:
+      cluster_id: UUID
+      cluster_name: string
+
+  - name: host_logs_uploaded
+    message: "Uploaded logs for host {host_name} cluster {cluster_id}"
+    event_type: host
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      host_id: UUID
+      host_name: string
+
+  - name: cluster_logs_uploaded
+    message: "Uploaded logs for cluster {cluster_id}"
+    event_type: cluster
+    severity: "info"
+    properties:
+      cluster_id: UUID
+
+  - name: host_status_updated
+    message: "Host {host_name}: updated status from '{src_status}' to '{new_status}' {status_info}"
+    event_type: host
+    severity: "info"
+    properties:
+      cluster_id: UUID
+      host_id: UUID
+      host_name: string
+      src_status: string
+      new_status: string
+      status_info: string
+      severity: string
```

Comment:
let's add documentation in commit message (the first commit) for the severity field which is allowed to override the event.severity field.

Reply:
solved